e2e-tests/tfm/cypress/e2e/journeys/case-amendments/facility-end-date/feature-flag-disabled/gef-deal-expect-no-facility-end-date-features.spec.js

```diff
@@ -0,0 +1,85 @@
+import relative from '../../../../relativeURL';
+import facilityPage from '../../../../pages/facilityPage';
+import { ADMIN, BANK1_MAKER1, PIM_USER_1, T1_USER_1 } from '../../../../../../../e2e-fixtures';
+import { MOCK_APPLICATION_AIN } from '../../../../../fixtures/mock-gef-deals';
+import { MOCK_FACILITY_ONE } from '../../../../../fixtures/mock-gef-facilities';
+import { DEAL_TYPE } from '../../../../../../../gef/cypress/fixtures/constants';
+import amendmentsPage from '../../../../pages/amendments/amendmentsPage';
+import dateConstants from '../../../../../../../e2e-fixtures/dateConstants';
+
+if (Cypress.env('FF_TFM_FACILITY_END_DATE_ENABLED') === 'false') {
+  context('Amendments - GEF deal does not display any Facility end date pages or fields when TFM feature flag disabled', () => {
+    let dealId;
+    let dealFacilities;
```

Comment:
is this one facility?

Reply:
A great spot, just the one so have renamed here and in my other new test files